templates/workspace_index/index.html.j2

```diff
@@ -24,4 +24,8 @@
         </li>
     {% endfor %}
 </ul>
-{% endblock %}
\ No newline at end of file
+
+{% if page.docs_content %}
+    {{ page.docs_content|safe }}
```

Comment:
I'm assuming the `safe` filter is fine here because the content is totally under the repo owner's control.

Reply:
yep, should be